pythran/pythonic/numpy/stack.hpp

```diff
@@ -14,6 +15,9 @@ namespace numpy
                  types::array<long, ArraySequence::value_type::value + 1>>
   stack(ArraySequence const &args, long axis)
   {
+    if(builtins::len(args) == 0)
+        throw pythonic::types::ValueError("need at least one array to stack");
+
```

Comment:
Add a test case? To test that an exception is raised, unittest provides `assertRaises`, e.g. in `pythran/tests/test_conversion.py`

Reply:
Done, but I'm not sure whether it's done right... I'm not sure how to verify that the test actually tests something and succeeds! (I get a pass, but I'm not 100% sure about all this)